### PR TITLE
Add script to control version bump, and syncronize versions to 3.0.0

### LINKED ISSUE #527: Versioning of subprojects
## overview

It should be possible to programatically check the version of any subproject of this repo. For example, the `opentrons` module of the api subproject should include a `__version__` member that can be checked to compare to the app version, etc. These numbers should change, at a minimum on new releases but preferably on every commit in order to verify exactly what version of code is being run at any given time.

## behavior

**Current behavior**: If version numbers exist, they are not updated or kept in sync across subprojects

**Desired behavior**: Version numbers for all subprojects should be incremented together, in whatever way makes sense for the language of each subproject, and should be available to check during runtime.

### PR DESCRIPTION
**Pull request by btmorr**
## overview

Adds a script to the repo for controlling the versions of sub-projects. Currently does not smart-detect subprojects, so `scripts/bump_version.py` will have to be modified to add any new sub-projects in the future. Also, unify all sub-project versions to 3.0.0

Fixes #527 

## changelog

- (feature) Add version bump script to repo and set all version numbers
- (chore) Remove versioneer (setup.py now writes correct _version.py file)

## Review requests

- Make sure that `cd api && pip install . && python -c "import opentrons; print(opentrons.__version__)"` prints `3.0.0`
- Make sure version numbers in JS projects report 3.0.0

### CODE REVIEW

File: version.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (mcous) [inline version.json]: For the purposes of playing nicely with some good javascript tooling for monorepo management, would we be opposed to naming this file `package.json` and putting `version` as a key? For example (to be further detailed in upcoming RFC ticket), we could put the list of all projects in this `package.json` as well (see [yarn workspaces](https://yarnpkg.com/lang/en/docs/workspaces/))

File: version.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "major": 3,
+  "minor": 0,
+  "bugfix": 0
```
Comment (mcous) [inline version.json]: (No change necessary, just something to chew on). I like having the version broken up, but I wonder if not doing a simple string prevents us from having the ability to do prerelease versions as allowed by [semver](https://semver.org/), e.g. `3.0.0-beta.1`. I know there are plenty of libraries on `npm` for processing version strings to get `major`, `minor`, and `patch`, and I imagine there are at least a few in python-land as well

Comment (btmorr) [reply version.json]: My thought was we could add a separate "tag" field that would append an arbitrary string to the end with "-" as a separator. I don't care strongly either way tho.

File: version.json (hunk 1 above)
Comment (btmorr) [reply version.json]: I'm fine with that. Will make the change.

**Pull request merged**